lib/broccoli/ember-app.js

```diff
@@ -263,20 +276,14 @@ EmberApp.prototype._initVendorFiles = function() {
     'jquery.js': this.bowerDirectory + '/jquery/dist/jquery.js',
     'handlebars.js': handlebarsVendorFiles,
     'ember.js': {
-      development: defaultDevelopmentEmber,
-      production:  this.bowerDirectory + '/ember/ember.prod.js'
+      development: developmentEmber,
+      production: productionEmber,
     },
     'ember-testing.js': [
       this.bowerDirectory + '/ember/ember-testing.js',
       { type: 'test' }
     ],
-    'app-shims.js': [
-      this.bowerDirectory + '/ember-cli-shims/app-shims.js', {
-        exports: {
-          ember: ['default']
-        }
-      }
-    ],
+    'app-shims.js': emberShims,
```

Comment:
why is it still mentioned in http://ember-cli.com/user-guide/#standard-amd-asset then? or am I confusing two totally unrelated things here?